README.md

```diff
@@ -11,23 +11,23 @@ The [MySQL](https://www.mysql.com/) operator provides an open-source relational
 ## Deployment instructions
 
 
-Create a Juju model for your operators, say "mysql"
+Create a Juju model for your operators, say "mysql-k8s"
 
 ```bash
-$ juju add-model mysql
+$ juju add-model mysql-k8s
```

Comment:
I think this needs to move into `CONTRIBUTING.md` or `INTEGRATION.md`. @jlounder?